float64 latitude
float64 longitude
float64 altitude
---
bool success
float32 x
float32 y
float32 z